Cargo.toml

```diff
@@ -131,3 +131,6 @@ proc-macro2 = "1.0"
 quote = "1.0"
 syn = "2.0"
 trybuild = "1.0"
+
+vergen = { version = "9.0.0", features = ["build", "rustc"] }
+cargo_metadata = "0.18.1"
```

Comment:
These should be put in the main dependencies group, not at the end of the file (and ordered alphabetically for consistency).